src/modules/asab-config/ConfigContainers/ConfigContainer.js

```diff
@@ -13,60 +13,204 @@ import { useTranslation } from 'react-i18next';
+			sortedData = sortedData.sort();
+			setTypeList(sortedData);
+			// TODO: validate responses which are not 200
+		}
+		catch(e) {
+			console.error(e);
+			App.addAlert("warning", t(`ASABConfig|Unable to get data for tree menu`));
+			return;
+		}
+	}
+
+	// Obtain the list of configs parsed to the type key
+	const getTree = async () => {
+		let tree = await Promise.all(typeList.map(t => getConfigs(t)));
+		setTreeList(tree);
+	}
+
+
+	const getConfigs = async (typeId) => {
+		let tree = {};
+		try {
+			let response = await ASABConfigAPI.get("/config/" + typeId);
+			if (response.data.result == 'OK'){
+				// Sort data
+				let sortedData = response.data.data;
+				if (sortedData != undefined) {
+					sortedData = sortedData.sort();
+					tree[typeId] = sortedData;
+				}
+			}
+			return tree;
+		}
+		catch(e) {
+			console.error(e);
+			App.addAlert("warning", t(`ASABConfig|Unable to get schema. Try to reload the page`, {type: typeId}));
+			return;
+		}
+	}
+
+	// Handle treeList to obtain the structure to render the tree
+	const getChart = () => {
+		let dataChart = [];
+		Object.values(treeList).map((element, idx) => {
+			addTreeStructure(element, dataChart);
+		});
+		let nodes = [];
+		dataChart.map(node => {
+			nodes.push(node.key)
+		});
+		setOpenNodes(nodes);
+		setTreeData(dataChart);
+	}
+
+
+	const addTreeStructure = (element, dataChart) => {
+		if (typeof element === 'object' && element !== null) {
+			Object.keys(element).map((key) => {
+				var obj = {
+					type: "folder", // this is not needed yet, but it might be useful for icons
+					key: key,
+					label: key,
+					nodes: []
+				};
+				dataChart.push(obj);
+
+				var index = dataChart.indexOf(obj);
+				if (element[key] != undefined) {
+					element[key].map((e) => {
+						if (typeof e === "object" && e !== null) {
+							addTreeStructure(e, dataChart[index].nodes);
+						} else if (typeof e === "string" && e !== null) {
+							var strObj = {
+								type: "file", // this is not needed yet, but it might be useful for icons
+								key: e,
+								label: e
+							};
+							dataChart[index].nodes.push(strObj);
+						}
+					})
+				}
+			})
+		} else if (element !== undefined) {
+			dataChart.push(
+				{
+					type: "file", // this is not needed yet, but it might be useful for icons
+					key: element,
+					label: element,
+				}
+			);
+		}
+	}
+
+
+	// Render function
 	return (
 		<Container fluid className="animated fadeIn flex mt-0 pr-0 pl-0 pt-0 config-container">
 			<Row className="config-row">
-				<Col sm="2" className="pr-0 bcg-column">
+				<Col sm="3" className="pr-0 bcg-column">
 					<TreeViewComponent
+						serviceURL={serviceURL}
+						openNodes={openNodes}
+						treeData={treeData}
+						setTreeData={setTreeData}
+						setChosenPanel={setChosenPanel}
 						app={props.app}
 						configCreated={props.config_created}
 						configRemoved={props.config_removed}
+						configImported={props.config_imported}
```

Comment:
@Aringoaway you dont need to have it here, since you dont use it in TreeViewComponent